<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointWidget</class>
 <widget class="QWidget" name="PointWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Survey Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSurveyId"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxClasses">
     <property name="title">
      <string>Classes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidgetClasses">
        <column>
         <property name="text">
          <string>Class</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonRemoveClass">
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Delete class.</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="../icons/ddg.qrc">
            <normaloff>:/icons/delete.svg</normaloff>:/icons/delete.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddClass">
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1000</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add class.</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../icons/ddg.qrc">
            <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSpinBox" name="spinBoxPointRadius">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Point Radius</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="checkBoxDisplayPoints">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Display</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spinBoxGrid">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Grid Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QCheckBox" name="checkBoxDisplayGrid">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Display</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Summary</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeView" name="treeView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonCounts">
            <property name="text">
             <string>Counts</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonPoints">
            <property name="text">
             <string>Points</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonChips">
            <property name="text">
             <string>Chips</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButtonExport">
          <property name="toolTip">
           <string>Export</string>
          </property>
          <property name="text">
           <string>Export</string>
          </property>
          <property name="icon">
           <iconset resource="../icons/ddg.qrc">
            <normaloff>:/icons/export.svg</normaloff>:/icons/export.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonLoadPoints">
          <property name="toolTip">
           <string>Load point file.</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
          <property name="icon">
           <iconset resource="../icons/ddg.qrc">
            <normaloff>:/icons/load.svg</normaloff>:/icons/load.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonImport">
          <property name="toolTip">
           <string>Import classes and custom field from existing point file.</string>
          </property>
          <property name="text">
           <string>Import</string>
          </property>
          <property name="icon">
           <iconset resource="../icons/ddg.qrc">
            <normaloff>:/icons/import.svg</normaloff>:/icons/import.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButtonReset">
          <property name="toolTip">
           <string>Reset point file.</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset resource="../icons/ddg.qrc">
            <normaloff>:/icons/reset.svg</normaloff>:/icons/reset.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButtonSave">
          <property name="toolTip">
           <string>Save point file.</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
          <property name="icon">
           <iconset resource="../icons/ddg.qrc">
            <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/ddg.qrc"/>
 </resources>
 <connections/>
</ui>
